src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/VideoLinkAppointment.kt

```diff
@@ -25,12 +25,15 @@ data class VideoLinkAppointment(
   @GeneratedValue(strategy = GenerationType.IDENTITY) val id: Long? = null,
   val bookingId: Long,
   val appointmentId: Long,
-  val court: String,
+  val court: String? = null,
+  val courtId: String? = null,
 
   @Enumerated(EnumType.STRING)
   val hearingType: HearingType,
 
   @CreatedBy
   var createdByUsername: String? = null,
   val madeByTheCourt: Boolean? = true
-)
+) {
+  fun chooseCourtName(): String = this.court ?: this.courtId ?: "Unknown"
```

Comment:
Is this going to be expanded on in the future? It will at some point have to always return a real court name

Reply:
In this long term this will become courtId -> court name lookup.  The source for this will probably be the courts register service.